src/components/attendee/VideoComms/hooks.tsx

```diff
@@ -0,0 +1,9 @@
+import { useContext } from "react";
+
+import { VideoCommsContext } from "./VideoCommsProvider";
+
+export const useVideoComms = () => {
+  // This might seem a little pointless. However, we expect that in the future
+  // this hook might do a little bit more than just get the context.
```

Comment:
[NOTE]
Anything that can be added to this hook will in all probability work added to the context.
The only thing this hook would be useful for is local state for a component.